src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/ampSrcBuilder.js

```diff
@@ -0,0 +1,11 @@
+import Url from 'url-parse';
+
+const ampSrcBuilder = href => {
+  const ampSrcPrefix = 'https://news.files.bbci.co.uk';
```

Comment:
Do we need to add a variant for `https://news.test.files.bbci.co.uk`?

Reply:
Yeah, we should definitely support the test variant, we should add this to our .env and vary by environment: https://github.com/bbc/simorgh/blob/latest/envConfig/test.env#L7